integration-tests/api/__tests__/admin/price-list.js

```diff
@@ -1244,6 +1244,29 @@ describe("/admin/price-lists", () => {
         expect.objectContaining({ id: "test-prod-2" }),
       ])
     })
+
+    it("lists products using free text search", async () => {
+      const api = useApi()
+
+      const response = await api
+        .get(`/admin/price-lists/test-list/products?q=MedusaHeadphones`, {
```

Comment:
suggestion: we should probably also try with half the string in order to validate that it function when ii si a partial match. what do you think?

Reply:
here we could remove the `%` at the query level and the tests would still work, wdyt?